sns_ik_examples/src/ik_tests.cpp

```diff
@@ -207,11 +231,14 @@ void test(ros::NodeHandle& nh, double num_samples_pos, double num_samples_vel,
   for (uint i=0; i < num_samples_pos; i++) {
     fk_solver.JntToCart(JointList[i],end_effector_pose);
     double elapsed = 0;
+    if (closePositionSeed)
+      q = JointDeltaSeed[i];
+    else if (i == 0 || !randomPositionSeed)
+      q = nominal;
+    else // "random" seed
+      q = JointList[i-1];
```

Comment:
It would be more robust if you defined the seed above and only used the seedVector in the three position solvers without copying this logic. The seedVector would would hold different values depending on the configuration parameter.